_benchmark/user-guide/finetine-workloads.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Finetuning custom workloads
+nav_order: 12
+parent: User guide
+---
+
+# Finetuning custom workloads
+
+While custom workloads can help make benchmarks more specific to your application's needs, they sometimes require additional work to make sure they closely resemble a production cluster.
+
+You can finetune your custom workloads to more closely match your benchmarking needs by using the 'create-workload' feature. `create-workload` can extract documents from all indexes or specific indexes selected by the user. The `create-workload` feature understands that all workloads have unique characteristics which allows you to finetune workload runs to your specific needs.
+
+## Characteristics to consider 
+
+When beginning to use `create-workload` to finetune a custom workload, consider the following characteristics of the workload:
+
+1. **Queries** - Consider what kinds of documents and indexes the queries are targeting and what fields the queries call.
+2. **Shard Size** - Match the shard size of the workload with the shard size in your cluster, or else the benchmark will not simulate the behavior of your application. Lucene operates on shard sizes and does not have a concept of indexes. Calculate the shard size for any index that you want in the custom workload.
+3. **Shard count** - Size the shard count according to how you want to size the workload and improve query performance. As each  use case is different, you can determine the shard count in two ways:
+    1. Divide the ideal index size with the shard size found in step 2.
+    2. Multiply the ideal number of shards with the shard size found in step 2.
+4. **Decide how many docs to extract (Recommended)** - Now that the shard size is set and the number of shards needed in the final index decided, you can determine how many docs you want to extract. In some cases, users aren’t interested in retrieving the entire document corpus from an index since the corpus might be too big. Instead, you might want to generate a smaller corpus. However, the corpus of the index generated should be *representative* of the index in the production workload. In other words, it should contain documents from across the index and not only from a certain area, for example, the first half of the index or the last third of the index. To decide how many docs to extract:
+    1. Multiply the number of shards with the shard size. Since every document is created unequally, add a buffer to the number, an arbitrary amount of additional documents. The buffer provides cover in situations when the number of docs extracted might come out under the expected amount, which does not help retain your shard size. The goal is for the shard size to be under the number of documents extracted. 
```

Comment:
```suggestion
    1. Multiply the number of shards by the shard size. Because every document is created unequally, add a buffer to the number---an arbitrary number of additional documents. The buffer provides assistance when the number of documents extracted is lower than the expected number, which will not help to retain your shard size. The shard size should be lower than the number of documents extracted. 
```